src/converters/toZigbee.ts

```diff
@@ -243,8 +243,18 @@ const converters2 = {
             utils.assertObject(value, key);
             const payload = value.payload !== undefined ? value.payload : {};
             utils.assertEndpoint(entity);
-            await entity.zclCommand(value.cluster, value.command, payload, value.options !== undefined ? value.options : {});
-            logger.info(`Invoked ZCL command ${value.cluster}.${value.command} with payload '${JSON.stringify(payload)}'`, NS);
+            await entity.zclCommand(
+                value.cluster,
+                value.command,
+                payload,
+                value.options !== undefined ? value.options : {},
+                value.logPayload ?? {},
```

Comment:
```suggestion
                value.log_payload ?? {},
```

Could you use snake case here? (same for the others)